test/scripts/test_disable_statements_check.py

```
@@ -0,0 +1,359 @@
+
+    def test_check_files_multiple(self):
+        """Test checking multiple files."""
+        file1 = self._create_temp_file(
+            "file1.ts", "// biome-ignore lint: temp\nconst x = 1;"
+        )
+        file2 = self._create_temp_file(
+            "file2.ts", "// @ts-ignore\nconst y = 2;"
+        )
+        violations = self.checker.check_files([file1, file2], repo="api")
+        # Should find violations in both files
+        self.assertGreaterEqual(len(violations), 2)
```

Ruff (0.14.10)

Use a regular `assert` instead of unittest-style `assertGreaterEqual`

Replace `assertGreaterEqual(...)` with `assert ...`

(PT009)